packages/light-apps/src/IdentityHeader/IdentityHeader.tsx

```diff
@@ -138,9 +138,8 @@ export class IdentityHeader extends React.PureComponent<Props, State> {
 
   handleChangeCurrentAccount = (account: string) => {
     const { history } = this.props;
-    const currentLocation = this.getCurrentLocation();
 
-    history.push(`/${currentLocation}/${account}`);
+    history.push(`/identity/${account}`);
```

Comment:
I think it's annoying to have the the UI redirect to identity after changing account from the transfer app. Doubly so that there's different behavior between changing the sender account in transfer account and changing the current account in the header.